src/php/Evaluator/MethodNodeEvaluator/Contract/RouterTest.php

```diff
@@ -0,0 +1,294 @@
+<?php
+
+namespace Viamo\Floip\Evaluator\MethodNodeEvaluator\Contract;
+
+use Viamo\Floip\Evaluator\MethodNodeEvaluator\Contract\EvaluatesMethods;
+use Viamo\Floip\Evaluator\MethodNodeEvaluator\Contract\TestResult;
+
+interface RouterTest extends EvaluatesMethods
+{
+    const PUNCTUATION=',:;!?.-';
```

Comment:
Are you purposely omitting dashes, brackets, braces, parentheses, quotes, and ellipsis?

Reply:
Following existing RapidPro punctuation rules.